test/Microsoft.AspNet.Razor.Runtime.Test/TagHelpers/TagHelperExecutionContextTest.cs

```diff
@@ -257,7 +259,7 @@ public void AddMinimizedHtmlAttribute_MaintainsHTMLAttributes()
         public void AddMinimizedHtmlAttribute_MaintainsHTMLAttributes_SomeMinimized()
         {
             // Arrange
-            var executionContext = new TagHelperExecutionContext("input", selfClosing: true);
+            var executionContext = new TagHelperExecutionContext("input", tagMode: TagMode.SelfClosing);
```

Comment:
Updated the other instance in this test class to go from `SelfClosing` => `StartTagOnly`. For other test classes I added specific tests that utilize `StartTagOnly`.